databuilder/file_formats/arrow.py

```diff
@@ -4,7 +4,6 @@
 import pyarrow
 
 from databuilder.file_formats.validation import ValidationError, validate_headers
-from databuilder.query_model.table_schema import Column, TableSchema
```

Comment:
This change is part of 9e477846. What should I expect from this commit? How should I review it?